### DISCUSSION TITLE
How turnoff Issuer of Patient ID

### DISCUSSION THREAD

**Discussion by itcthienkhiem** [dcm4che/dcm4chee-arc-light#4513] [General]
When entering the configuration screen of dcm4chee, I do not see the Issuer of Patient ID field. Currently I want to turn this field off to not allow this field to be recorded when modality send to PACS
So how should we configure to ignore this field?

**Comment by vrindanayak** [comment]
> Currently I want to turn this field off to not allow this field to be recorded when modality send to PACS

Issuer of Patient ID is recommended to keep the patient identifiers unique in the archive and to avoid issues in patient management / mix up of studies of different patients
